tests/upgrade/test_existence.py

```diff
@@ -0,0 +1,139 @@
+"""Upgrade TestSuite for validating Satellite entities existence and
```

Comment:
Moved this test to the separate directory here:  upgrade_tests/test_existence.py